.github/workflows/comment-on-pr.yaml

```diff
@@ -0,0 +1,13 @@
+name: Remind about milestones and labels
+on: pull_request
+jobs:
+  comment:
+    name: Remind labels and milestones
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
I don't think this looks like it would be right. Should it be actions/pr? I've got no idea.

Reply:
Nevermind looks like this is needed to have access to the template.md in the repo.